Core/Preview.lua

```diff
@@ -57,8 +57,12 @@ end
 
 local function resizeOnMouseDown(self)
 	local f = self:GetParent();
-	f:SetMinResize(335,385);
-	f:SetMaxResize(GetScreenWidth(),GetScreenHeight());
+	if f.SetResizeBounds then
+		f:SetResizeBounds(335, 385, GetScreenWidth(), GetScreenHeight());
+	else
+		f:SetMinResize(335,385);
+		f:SetMaxResize(GetScreenWidth(), GetScreenHeight());
+	end
```

Comment:
I'm guessing the `if` is for backwards compatibility? I don't think there's much sense in having that here, as no such effort has been done for the the rest of the codebase.

Reply:
In theory yes. I wasn't sure if this was something you wanted or not, if so I can look into adding more compatibility checks.

Reply:
To be clear, I didn't mean that *you* had made no such effort! But I would rather leave this out, for now.